frontend/public/components/monitoring/_monitoring.scss

```diff
@@ -1,3 +1,33 @@
+.co-m-pane__heading--monitoring-dashboards {
+  justify-content: flex-start;
+}
+
+.monitoring-dashboards-dropdown {
```

Comment:
We should follow BEM naming for these classes

Reply:
👍